content/en/apps/reference/app-settings/_index.md

```diff
@@ -224,3 +224,5 @@ matches "mary", "Mary", "john", "John", "JOhN", etc. Not "maryjohn"
 | `uniqueWithin(*fields, time_period)` | Same as `unique` but the last argument is the time period in which to search. Eg `uniqueWithin('form', 'patient_id', '1 week')` checks that the same form wasn't submitted for this patient in the past week. |
 | `exists(form_id, field)` | Returns `true` if a report matches the `form_id` and value for `field`. This is useful to check that a patient was registered for a service before reporting about it. Eg `exists('REG', 'patient_id')` checks that a `REG` form was already submitted for a patient. As of 2.12 most uses of this function are obsolete because checking for a valid `patient_id` is done automatically by the `accept_patient_report` transition using `registration_not_found` in the `messages.event_type`. |
 | `isISOWeek(weekFieldName[, yearFieldName])` | Returns `true` if the current report has a week value that is less or equal to the number of ISO weeks of the current year or the year value of the same report. The first parameter is the field name for the week and the second parameter is the field name for the year: `isISOWeek('week', 'year')`. If the second parameter is not specified, then the current year is used: `isISOWeek('week')`. |
+| `isAfter(duration)` | Returns `true` if the date property comes after today plus the duration. For this to work, the property should be a date type. e.g. `isAfter('2 weeks')` checks that the date property is at least two weeks in future from today. Negative values are also supported. |
```

Comment:
Looks good. If you feel like (sounds like a silly nitpick), you could add a `the`  before `future` (`in future`), but ... :)